frontend/webadmin/modules/frontend/src/main/java/org/ovirt/engine/ui/frontend/server/dashboard/DashboardDataServlet.java

```diff
@@ -362,6 +369,25 @@ private GlobalUtilization lookupGlobalUtilization() throws DashboardDataExceptio
         return utilization;
     }
 
+    /**
+     * Return the configured Grafana base URL or null if either Grafana is not configured
+     * or the config key is empty.
+     */
+    private String lookupEngineGrafanaBaseUrl(EngineLocalConfig config) {
+        String grafanaBaseUrlVal;
+
+        try {
+            grafanaBaseUrlVal = config.getEngineGrafanaBaseUrl();
+            if (StringUtils.isNotBlank(config.getEngineGrafanaFqdn()) && StringUtils.isNotBlank(grafanaBaseUrlVal)) {
```

Comment:
It seems wrong. Consider change to the following.

```suggestion
if (StringUtils.isEmpty(config.getEngineGrafanaFqdn()) || StringUtils.isEmpty(grafanaBaseUrlVal)) {
```

Reply:
Good catch - thanks.